protocol/testutil/keeper/prices.go

```diff
@@ -94,18 +100,86 @@ func createPricesKeeper(
 			lib.GovModuleAddress.String(),
 		},
 		revShareKeeper,
+		marketMapKeeper,
 	)
 
 	return k, storeKey, indexPriceCache, mockTimeProvider
 }
 
+// Convert MarketParams into MarketMap markets and create them in the MarketMap keeper.
+func CreateMarketInMarketMapFromParams(
```

Comment:
```suggestion
func CreateMarketsInMarketMapFromParams(
```
This function creates multiple markets right?